Azure/Azure-AppServices-Diagnostics-Portal
# Adding a CoachMark specific to Web App (Linux)

## Overview
Adding a Resiliency Score Report CoachMark specific for Web App (Linux) to help bring attention as if they had already seen the Resiliency Score Report for Web App (Windows), it won't show the coachMark anymore.

## Related Work Item
N/A

## Type of change



## Solution description
Adding a Resiliency Score Report CoachMark specific for Web App (Linux).
Adding a Web App (Linux) specific Teaching Bubble Headline and Text

## How Can This Be Tested? <span>&#128269;</span>
In the Azure Portal go to a Web App (Linux) in Standard or above, click in Diagnose and solve problems\Availability and Performance and the very first time, you should be able to see a teaching bubble pointing at Get Resiliency Score Report:
 
![image](https://user-images.githubusercontent.com/7376302/218888422-7d5e7680-2bb7-4f08-8623-c7c67a19055f.png)
Hovering over it shows the following text:
![image](https://user-images.githubusercontent.com/7376302/218888454-74cb843b-5b23-4921-95bc-fb550fa9617f.png)


## Checklist <span>&#9989;</span>

## Screenshots Before Fix (if appropriate):

## Screenshots After Fix (if appropriate):

## Extra Notes
